leptonai/cli/deployment.py

```diff
@@ -714,7 +714,7 @@ def create(
     try:
         logger.trace(f"deployment_template:\n{deployment_template}")
         template_envs = deployment_template.env or {}
-        env_list = env or []
+        env_list = list(env) or []
```

Comment:
Thanks - let's fix the following ones as well for secret and mount. I think the passed-in params are tuple instead of list.